src/server/handlers/github-auth.js

```diff
@@ -146,6 +146,6 @@ export const logout = (req, res, next) => {
       return next(new Error('Failed to log out.'));
     }
     // FIXME redirect to page that initiated the sign in request
-    res.redirect('/');
+    res.redirect('https://snapcraft.io/logout');
```

Comment:
While I kind of understand why we want do to it, I'm afraid it may lead to some unexpected redirect issues or raise conditions.

Reply:
Yeah, I want to catch this as part of snapcraft.io. I think it could be an issue, but the other worry is the inconsistency of 'logout' not actually logging out of everything and confusing the user.